### PR TITLE
Update quickstart with cargo stylus check timing and Docker requirements

### PR DESCRIPTION
**Pull request by mahsamoosavi**

### CODE REVIEW

File: arbitrum-docs/stylus/stylus-quickstart.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ Compressed WASM size: 3 KB
 Program succeeded Stylus onchain activation checks with Stylus version: 1
 ```
 
+Note that running `cargo stylus check` may take a few minutes, especially on a fresh build. If you're running the check or deploy commands, a message at the start will inform you that this process might take some time. For faster checks, you can use the `--no-verify` option. Additionally, please ensure that Docker is up and running as it's required for these commands to execute properly.
```
Comment (anegg0) [inline arbitrum-docs/stylus/stylus-quickstart.md:154]:
```suggestion
Note that running `cargo stylus check` may take a few minutes, especially on a fresh build. If you're running the check or deploy commands, a message at the start will inform you that this process might take some time. For faster checks, you can use the `--no-verify` option. Also, please ensure that Docker is up and running, as these commands are required to be executed properly.
```

**Review by anegg0 — CHANGES_REQUESTED** [review]
Just a nit, then let's merge!

**Review by TucksonDev — APPROVED** [review]
LGTM

**Review by anegg0 — APPROVED** [review]
LGTM

**Pull request merged**